clang/test/Parser/cxx1z-decomposition.cpp

```diff
@@ -136,7 +136,7 @@ namespace MultiDeclarator {
 
 namespace Template {
   int n[3];
-  // FIXME: There's no actual rule against this...
+  // Structured binding template is not allowed.
   template<typename T> auto [a, b, c] = n; // expected-error {{decomposition declaration template not supported}}
```

Comment:
While we are there, can we change the diagnostic to `decomposition declaration cannot be a template` or something like that, not to give the impression it's a clang limitation?